### DISCUSSION TITLE
zoom download zoom-rec-dl

### DISCUSSION THREAD

**Discussion by m4motiv8** [hyunbinseo/zoom-rec-dl#5] [Q&A] [answered]
Hi, figured our step 4, however now i get this:

C:\Users\Mac Laubscher\Downloads\zoom-rec-dl-main>node index.mjs
(node:29084) ExperimentalWarning: Importing JSON modules is an experimental feature and might change at any time
(Use `node --trace-warnings ...` to show where the warning was created)
undefined:2
https://us02web.zoom.us/rec/share/rIzRO5nGR_terx9v7qkatpY1bZA5pWwnXP6Dx3b0eRAC9niOrczUfESJbnLsKO8w.P16T1HUO8sgnlBgB
^

SyntaxError: C:\Users\Mac Laubscher\Downloads\zoom-rec-dl-main\urls.json: Unexpected token h in JSON at position 2
    at parse (<anonymous>)
    at ESMLoader.jsonStrategy (node:internal/modules/esm/translators:299:21)
    at ESMLoader.moduleProvider (node:internal/modules/esm/loader:468:14)

Node.js v18.13.0

C:\Users\Mac Laubscher\Downloads\zoom-rec-dl-main>

**Comment by m4motiv8** [comment]
or this: (node:10892) ExperimentalWarning: Importing JSON modules is an experimental feature and might change at any time
(Use `node --trace-warnings ...` to show where the warning was created)
undefined:1
(https://us02web.zoom.us/rec/share/rIzRO5nGR_terx9v7qkatpY1bZA5pWwnXP6Dx3b0eRAC9niOrczUfESJbnLsKO8w.P16T1HUO8sgnlBgB)
^

SyntaxError: C:\Users\Mac Laubscher\Downloads\zoom-rec-dl-main\urls.json: Unexpected token ( in JSON at position 0
    at parse (<anonymous>)
    at ESMLoader.jsonStrategy (node:internal/modules/esm/translators:299:21)
    at ESMLoader.moduleProvider (node:internal/modules/esm/loader:468:14)

**Comment by hyunbinseo** [comment, marked as answer]
Replace the `urls.json` file's content with the following code block.

```json
[
  "https://us02web.zoom.us/rec/share/rIzRO5nGR_terx9v7qkatpY1bZA5pWwnXP6Dx3b0eRAC9niOrczUfESJbnLsKO8w.P16T1HUO8sgnlBgB"
]
```
